jkamz/Questioner
# #162982934 A user should be able to view their profile page

### What does this PR do?
This PR creates the users profile page

### Description of the tasks to be completed
The tasks for this PR include:

- Create the profile section

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ft-user-profile-162982934`

Navigate to UI/templates folder and open profile.html on browser

### What are the relevant Pivotal Tracker stories?
#162982934

### Screenshots

![profile](https://user-images.githubusercontent.com/11965186/50751691-eed87d80-125b-11e9-8ead-51c5c4c7d175.png)


Mobile Site

![profilemob](https://user-images.githubusercontent.com/11965186/50751698-f5ff8b80-125b-11e9-9d44-147499f74831.png)
